src/extension.ts

```diff
@@ -236,6 +236,11 @@ export namespace Openshift {
             const tls = checkTls ? "https://" : "http://";
             opn(`${tls}${hostName.stdout}`);
         };
+
+        export const openshiftConsole = async function (odo: odoctl.Odo) {
```

Comment:
since it is called from context menu on cluster node, there should be `context` parameter in the function. 
context is OpenShiftObject instance, therefore you do not need to search for clusters, just do
`opn(context.getName());`